ultrasonic/src/main/kotlin/org/moire/ultrasonic/playback/AutoMediaBrowserCallback.kt

```diff
@@ -177,6 +206,104 @@ class AutoMediaBrowserCallback(var player: Player) :
         return onLoadChildren(parentId)
     }
 
+    override fun onCustomCommand(
+        session: MediaSession,
+        controller: MediaSession.ControllerInfo,
+        customCommand: SessionCommand,
+        args: Bundle
+    ): ListenableFuture<SessionResult> {
+
+        var customCommandFuture: ListenableFuture<SessionResult>? = null
+
+        when (customCommand.customAction) {
+            SESSION_CUSTOM_SET_RATING -> {
+                /*
+                * It is currently not possible to edit a MediaItem after creation so the isRated value
+                * is stored in the track.starred value
+                * See https://github.com/androidx/media/issues/33
+                */
```

Comment:
Yes I tried a lot using with parameter for rating but I kept bumping on issues with those custom Commands
I'll give it one last try when I go back to it

Reply:
I have to create a custom Command because using the DefaultMediaNotificationProvider requires it's actions to be custom Commands
https://github.com/androidx/media/blob/2c7201024ebdbfccf32e291f816fe426cc5540d8/libraries/session/src/main/java/androidx/media3/session/DefaultActionFactory.java#L91